doc/api/util.md

```diff
@@ -330,6 +330,24 @@ changes:
 The `util.inspect()` method returns a string representation of `object` that is
 primarily useful for debugging. Additional `options` may be passed that alter
 certain aspects of the formatted string.
+`util.inspect()` will use the constructor's name and/or `@@toStringTag` to make an
+identifiable tag for an inspected value.
+
+```js
+class Foo {
+  get [Symbol.toStringTag]() {
+    return 'bar';
+  }
+}
+
+class Bar {}
+
+const baz = Object.create(null, { [Symbol.toStringTag]: { value: 'foo' } });
+
+util.inspect(new Foo()); // 'Foo [bar] {}'
+util.inspect(new Bar()); // 'Bar {}'
+util.inspect(baz);       // '[foo] {}'
```

Comment:
Is there a test for this? Unless I'm mistaken in my reading of the code, `baz` would be inspected as `' [foo] {}'` rather than `'[foo] {}'` with the current iteration.

Reply:
I think `util.inspect` will also display the Symbol if it's not defined as `enumerable: false`, which is the case in this example.

Reply:
symbols aren't shown unless you pass true for the second arg
![](https://gc.gy/⚤.png)